edit-post/store/reducer.js

```diff
@@ -63,6 +68,17 @@ export const preferences = combineReducers( {
 
 		return state;
 	},
+	pinnedPluginItems( state = PREFERENCES_DEFAULTS.pinnedPluginItems, action ) {
+		if ( action.type === 'TOGGLE_PINNED_PLUGIN_ITEM' ) {
+			return {
+				...state,
+				[ action.pluginName ]: isUndefined( state[ action.pluginName ] ) ?
```

Comment:
Should we want to use [`Object#hasOwnProperty`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) here so we're not dealing with oddities with my aptly-named "constructor" plugin?

```
$ n_
n_ > var state = {};
undefined
n_ > var action = { pluginName: 'constructor' };
undefined
n_ > _.isUndefined( state[ action.pluginName ] );
false
```